backend/src/schema/types/type/User.gql

```diff
@@ -68,10 +68,11 @@ type User {
   RETURN COUNT(u) >= 1
   """
   )
-  isBlocked: Boolean! @cypher(
+  
+  blocked: Boolean! @cypher(
   statement: """
-  MATCH (this)<-[: BLOCKED]-(u: User { id: $cypherParams.currentUserId})
-  RETURN COUNT(u) >= 1
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-20T16:39:21Z" title="Monday, January 20th 2020, 5:39:21 pm +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -68,10 +68,11 @@ type User {
   RETURN COUNT(u) >= 1
   """
   )
+  
   isBlocked: Boolean! @cypher(
   statement: """
-  MATCH (this)<-[: BLOCKED]-(u: User { id: $cypherParams.currentUserId})
-  RETURN COUNT(u) >= 1
+  MATCH (this)<-[:BLOCKED]-(user:User {id: $cypherParams.currentUserId})
```


```suggestion
  MATCH (this)-[:BLOCKED]-(user:User {id: $cypherParams.currentUserId})
```
I would argue `BLOCKED` relationship is vice versa and then we can refactor the code below :point_down:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-20T16:49:56Z" title="Monday, January 20th 2020, 5:49:56 pm +01:00">Jan 20, 2020</time>_

---


I was working under the understanding that blocking someone means only that a user can't interact with my content... Therefore not reciprocal